linux/ca-certificates/debian/src/main/packaging/debian/changelog

```diff
@@ -1,3 +1,9 @@
+adoptium-ca-certificates (1.0.1-1) STABLE; urgency=medium
+
+  * Add Ubuntu Kinetic (22.10) to the list of supported Ubuntu releases.
+
+ -- Eclipse Adoptium Package Maintainers <temurin-dev@eclipse.org>  Tue, 06 Dec 2022 15:42:00 +0000
```

Comment:
update date to 19 Dec?

Reply:
Don’t think it’s really necessary, I’ve already pushed the updated CA Cert package so the date would be correct